warehouse/templates/index.html

```diff
@@ -15,16 +15,29 @@
 
 {% block title %}PyPI - the Python Package Index{% endblock %}
 
-{% macro project_snippet(release) -%}
-<div class="package-snippet">
-  <h3 class="package-snippet__title">
-    <a href="{{ request.route_path('packaging.project', name=release.project.normalized_name) }}">{{ release.project.name }}</a>
-    <span class="package-snippet__version">{{ release.version }}</span>
-  </h3>
-  <p class="package-snippet__description">{{ release.summary }}</p>
+{% set trending_projects_visible = namespace(total=20) %}
+{% set latest_releases_visible = namespace(total=20) %}
+
+{% macro project_snippet_latest(release, summary, index) -%}
+<div class="package-snippet hide-by-index-latest">
+  {{ project_snippet(release, summary, index)}}
+</div>
+{%- endmacro %}
+{% macro project_snippet_trending(release, summary, index) -%}
+<div class="package-snippet hide-by-index-trending">
+  {{ project_snippet(release, summary, index)}}
 </div>
 {%- endmacro %}
 
+{% macro project_snippet(release, summary, index) -%}
```

Comment:
do you need `summary` as an argument? seems you're retrieving it from `release` or defaulting anyway

Reply:
Nope, unnecessary, you are right